File: backend/Core/DTOs/PerfumeSuggestionRequestDto.cs
namespace Core.DTOs
{
    public sealed class PerfumeSuggestionRequestDto
    {
        public string Brand { get; init; } = null!;
        public string Name { get; init; } = null!;
        public string? Comment { get; init; }
        public string? ImageUrl { get; init; }
        public string? ImageBase64 { get; init; }
        public IReadOnlyList<string> Groups { get; init; } = [];
        public IReadOnlyList<NoteGroupDto> NoteGroups { get; init; } = [];
    }
}

File: mobile/src/app/models/perfumesuggestionrequest.dto.ts
import { NoteGroupDto } from "./notegroup.dto";

export interface PerfumeSuggestionRequestDto {
  brand: string;
  name: string;
  groups: string[];
  noteGroups: NoteGroupDto[];
  comment?: string;
  imageUrl?: string;
  imageBase64?: string;
}
